src/main/java/org/skife/jdbi/v2/SQLStatement.java

```diff
@@ -1329,6 +1329,11 @@ private String wrapLookup(String sql)
             timingCollector.collect(elapsedTime, getContext());
         }
         catch (SQLException e) {
+            try {
+                stmt.close();
+            } catch (SQLException e1) {
+                e1.printStackTrace(); // TODO: we have nowhere to log this, and suppressed exceptions were added in Java 7...
```

Comment:
slf4j is not a mandatory dependency, so cannot be used or you risk linking errors:

```
        <dependency>
            <groupId>org.slf4j</groupId>
            <artifactId>slf4j-api</artifactId>
            <optional>true</optional>
        </dependency>
```

:cry: